src/Microsoft.AspNetCore.Server.HttpSys/MessagePump.cs

```diff
@@ -56,14 +56,11 @@ public MessagePump(IOptions<HttpSysOptions> options, ILoggerFactory loggerFactor
 
             _processRequest = new Action<object>(ProcessRequestAsync);
             _maxAccepts = _options.MaxAccepts;
-            EnableResponseCaching = _options.EnableResponseCaching;
```

Comment:
What happens now if some sets HttpSysOptions.EnableResponseCaching? Is it ignored?